cmake/nanobind-config.cmake

```
@@ -4,20 +4,40 @@ if (NOT TARGET Python::Module)
   message(FATAL_ERROR "You must invoke 'find_package(Python COMPONENTS Interpreter Development REQUIRED)' prior to including nanobind.")
 endif()
 
-# Determine the Python extension suffix and stash in the CMake cache
-execute_process(
-  COMMAND "${Python_EXECUTABLE}" "-c"
-    "import sysconfig; print(sysconfig.get_config_var('EXT_SUFFIX'))"
-  RESULT_VARIABLE NB_SUFFIX_RET
-  OUTPUT_VARIABLE NB_SUFFIX
-  OUTPUT_STRIP_TRAILING_WHITESPACE)
-
-if (NB_SUFFIX_RET AND NOT NB_SUFFIX_RET EQUAL 0)
-  message(FATAL_ERROR "nanobind: Python sysconfig query to "
-    "find 'EXT_SUFFIX' property failed!")
+# Determine the right suffix for ordinary and stable ABI extensions
+if (Python_SOSABI)
```

```suggestion
if (DEFINED Python_SOSABI)
```

It'll be empty on Windows, which is false.

---

Thanks!